pkg/build/webhook/controller.go

```diff
@@ -20,7 +20,7 @@ type Plugin interface {
 	// - newly created build object or nil if default is to be created
 	// - information whether to trigger the build itself
 	// - eventual error.
-	Extract(buildCfg *buildapi.BuildConfig, secret, path string, req *http.Request) (*buildapi.SourceRevision, bool, error)
+	Extract(buildCfg *buildapi.BuildConfig, secret, path string, req *http.Request) (*buildapi.SourceRevision, []kapi.EnvVar, bool, error)
```

Comment:
@gabemontero @bparees why don't i take care of this in https://github.com/openshift/origin/pull/8580 ?